src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -15,64 +16,45 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// <summary>
     /// A TupleTypeSymbol represents a tuple type, such as (int, byte) or (int a, long b).
     /// </summary>
-    internal sealed class TupleTypeSymbol : NamedTypeSymbol
+    internal sealed class TupleTypeSymbol : WrapperNamedTypeSymbol
     {
-        private readonly NamedTypeSymbol _underlyingType;
-        private readonly ImmutableArray<TupleFieldSymbol> _fields;
-        private readonly bool _hasFriendlyNames;
+        private readonly ImmutableArray<Location> _locations;
+        private readonly ImmutableArray<Location> _elementLocations;
```

Comment:
Not sure what this means. It is indexed by tuple element position and gives a position for that element? A comment would be helpful.